docs/paper/dev/misc/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging your plugin
+
+Debugging your plugin is vital to being able to fix bugs and issues in your plugin. This page will cover some of the most common debugging techniques.
+
+### Randomly printing to the console
```

Comment:
```suggestion
### Printing to the console
```
I would also add a warning here to make sure it's immediately obvious this is not the ideal solution